sys/include/timex.h

```diff
@@ -22,7 +22,6 @@
 #define __TIMEX_H
 
 #include <stdint.h>
-#include <stdio.h>
```

Comment:
That's unfortunate... Well then let's make this PR a cleanup of other people's messes to, shalt we :-)